sw/device/silicon_creator/lib/sigverify/rsa_verify.c

```diff
@@ -169,9 +169,8 @@ static hardened_bool_t sigverify_use_sw_rsa_verify(lifecycle_state_t lc_state) {
   switch (launder32(lc_state)) {
```

Comment:
How about this? 

```c
static hardened_bool_t sigverify_use_sw_rsa_verify(void) {
  uint32_t sw_en  = otp_read32(
          OTP_CTRL_PARAM_CREATOR_SW_CFG_SIGVERIFY_RSA_MOD_EXP_IBEX_EN_OFFSET);
  if (launder32(sw_en) == kHardenedBoolFalse) {
    return kHardenedBoolFalse;
  }
  HARDENED_CHECK_NEQ(sw_en, kHardenedBoolFalse);
  return kHardenedBoolTrue;
}
```

Reply:
I think we can do one better and remove the `sigverify_use_sw_rsa_verify` function entirely, since we want the same behavior across all LC states.

Reply:
SGTM